CONTRIBUTING.md

```diff
@@ -1,14 +1,95 @@
 # Contributors guide
 
-## Markdown style
```

Comment:
Thank you for starting this @Park-Patrick! Is this markdown style section incorporated into the file you've started? It does need to be, and it probably makes sense to add it as a section that can be linked from the table of contents.

Some explanation of what the continuous integration is doing, what linting is and why it needs to be done would be really great. ✨

Reply:
right makes sense! 
also there is an markdown section but I'll add the link to the table of contents